packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/export/saved_objects_exporter.test.ts

```diff
@@ -247,7 +249,7 @@ describe('getSortedObjectsForExport()', () => {
 
       describe('>1k hits', () => {
         const firstMockHits = generateHits(1000, { sort: ['a', 'b'] });
-        const secondMockHits = generateHits(500);
+        const secondMockHits = generateHits(500, { idPrefix: 'second-hit-' });
```

Comment:
This test was generating multiple pages of SOs with duplicate ids. The fix performed in this PR also has the impact of removing duplicates from top level / initial objects. Note that this will never occur in production given we are removing duplicates from the top level exported objects anyway.